app/javascript/styles/mastodon/components.scss

```diff
@@ -3395,6 +3395,16 @@ $ui-header-height: 55px;
   border-color: $ui-highlight-color;
 }
 
+.react-toggle:hover:not(.react-toggle--disabled) .react-toggle-track {
+  background: darken($ui-primary-color, 5%);
+}
+
+.react-toggle.react-toggle--checked:hover:not(.react-toggle--disabled)
+  .react-toggle-track {
+  background: lighten($ui-highlight-color, 5%);
+}
+
+
```

Comment:
`prettier` isn't happy about this one. You should be able to have it fix it automatically by running `yarn run format`.

Reply:
Will do! Thank you!